src/eradiate/_config.py

```diff
@@ -8,6 +8,7 @@
 import enum
 import os.path
 import pathlib
+from importlib.metadata import version, PackageNotFoundError
```

Comment:
How about factoring the Mitsuba package import and version detection code out in the `kernel` subpackage? There is already a bit of logic here, we could have a small interface like:
```python
eradiate.kernel.FOUND_DRJIT_MODULE  # True iff drjit module can be imported
eradiate.kernel.FOUND_MITSUBA_MODULE  # True iff mitsuba module can be imported
eradiate.kernel.FOUND_ERADIATE_MITSUBA_PACKAGE  # True iff eradiate-mitsuba package can be imported
eradiate.kernel.ERADIATE_MITSUBA_VERSION  # String with kernel version number (raises if wrong Mitsuba is used)
# All variables set upon `import eradiate.kernel`; failures raise exceptions.
```

Reply:
Unless I am missing smth, I don't think we can opt out of having a mitsuba or drjit modules available in the current working environment. I don't really see the point of using an extra variable to tell if I found the module, if I raise when I can not find it anyway. Drjit and mitsuba availabilities are already tested in `kernel.__init__.py`. 

However, checking for the existence of the eradiate-mitsuba package, its version, the kernel version and the eradiate patch version can be outsourced in the `eradiate.kernel` module